test/SlowTests/Cluster/ClusterTransactionTests.cs

```diff
@@ -109,70 +109,80 @@ private string RandomString(int length)
         [InlineData(5)]
         public async Task CanPreformSeveralClusterTransactions(int numberOfNodes)
         {
```

Comment:
We should do this only on our server, so we need `UseLocalServer` here too, I think

Reply:
Not needed: `var leader = await CreateRaftClusterAndGetLeader(numberOfNodes);` is being used in this test (creates new servers)